tests/invalid_image/CMakeLists.txt

```diff
@@ -4,4 +4,5 @@
 add_executable(invalid_image main.cpp)
 target_link_libraries(invalid_image oehost)
 set_property(TARGET invalid_image PROPERTY POSITION_INDEPENDENT_CODE OFF)
+target_link_options(invalid_image PRIVATE -no-pie)
```

Comment:
Did this stem from a warning that position independent code is off?

Reply:
I should have given context on this: The test requires the binary to be non-PIE:
https://github.com/openenclave/openenclave/blob/6a5e7a16a243725592e63fe0097f03256301545e/tests/invalid_image/main.cpp#L10-L13
That's why it defines `POSITION_INDEPENDENT_CODE OFF`. On Ubuntu 24.04 with clang-14, cmake passes the flag to the compiler, but not to the linker. So I added it to make the test work again.